src/screens/gm/gameMasterScreen.cpp

```diff
@@ -321,7 +321,7 @@ void GameMasterScreen::update(float delta)
 
     if (targets.getTargets().size() == 1)
     {
-        selection_info["Position"] = string(targets.getTargets()[0]->getPosition().x, 0) + "," + string(targets.getTargets()[0]->getPosition().y, 0);
+        selection_info[trMark("gm_info", "Position")] = string(targets.getTargets()[0]->getPosition().x, 0) + "," + string(targets.getTargets()[0]->getPosition().y, 0);
```

Comment:
@muerteFR Any preference for a context name? Couldn't really remember which convention you went with...

Seeing as you made the bulk of the translation work, let me know if it's not in line with the rest.

Reply:
It seems very clear for me :-) Thanks for asked me :-)